install_guide_pi.md

```diff
@@ -19,39 +22,47 @@ Open raspi-config to enable SSH
 $ sudo raspi-config
 ```
 
-Select 7 advanced options and A4 SSH and enable ssh server
+Select Advanced Options 
+ SSH and enable ssh server
+ Hostname and give you pi a name
 
 Check IP-address
 ```sh
 $ ip a
 ```
 
-Pretty likely it is something like 192.168.1.x (can be found ~4th row from the bottom)
+Expect something like 192.168.1.x (can be found ~4th row from the bottom)
 
-Now you can either connect to raspberry with SSH (e.g. Putty from windows) or continue using current terminal
+Now you can either connect to raspberry with SSH (e.g. Putty from windows) or continue using current terminal.
 
-If you connect from SSH, use these login credentials
+Use these login credentials
 ```
 username: pi 
 password: raspberry
 ```
+--> 
 
 ### Update System
 
-Update package indexes and upgrade installed packages. This will take some minutes
+Update package indexes and upgrade installed packages. 
+apt-get is rather verbose and how long it takes depends on what needs to be updated.
 ```sh
-$ sudo apt-get update
-$ sudo apt-get upgrade
+$ sudo apt-get update &&  sudo apt-get dist-upgrade && echo +++ upgrade successful +++
```

Comment:
Remove && echo +++ install successful +++

Reply:
Prefer to leave echo success (which only outputs if return code is 0)
since apt-get stops quietly after many verbose messages. OK?